rfcs/build-system/05-folder-organization.md

```diff
@@ -0,0 +1,63 @@
+# RFC: Fluent UI Repo Folder Organization
+
+---
+
+Contributors: Fluent UI React Build team, @justSlone
+
+## Summary
+
+This propsoal covers 3 topics:
+1. organization of rfcs folder
+2. organization (or removal) of specs folder
+3. organization of packages folder (can be made later)
+
+Overall this RFC proposes that we organize our repo by NPM package names under the @fluentui namespace. 
+These names today are `react`, `react-components`, `react-northstar`, and `web-components`. Organizing by NPM package name, 
+while not necessarily ideal and may change over time, has the benefit of making it easy to follow from NPM to the appropriate source code in GitHub. 
+This also scales well if we add additional projects to the repo (as long as they are under the @fluentui namespace on NPM). 
+
+## Background
+
+Some useful background is that we have heard from contributors that the current organization of the rfcs and specs folders at the root of the 
+repo is confusing to contributors working on web-components. It is also confusing to anyone working in the repo. For instance, understanding if react-examples is v8 or vNext examples is not immediately clear and requires deeper inspection. 
+
+## Problem statement
+
+With the addition of vNext to the repo we now have 4 actual projects in our repo: 
+@fluentui/react, @fluentui/react-northstar, @fluentui/web-components, and more recently @fluentui/react-components.
+
+This leads to a fairly confusing and difficult to navigate repo. For instance, understanding if react-examples is v8 or vNext examples is not immediately clear and requires deeper inspection. Furthermore we have some folders we are actively using, such as rfcs and specs which could apply to any of the 4 projects, but in fact only really apply to @fluentui/react-components. This is potentially confusing to contributors visiting our repo, and generally a ramp-up headache. 
+
+## Detailed Design or Proposal
+
+1. Within the rfcs folder we should create a new subfolder called `react-components` and move all existing rfcs to this folder. 
+   - if needed additional `react`, `react-northstar`, or `web-components` RFCs can be created
+   - RFCs that apply to the whole repo can go in a folder called `general` (if anyone has a better name, let me know!)
+2. We should remove the specs folder. We should be keeping specs up to date under the relevant package folders. Associating specs with particular packages scales better in a large repo such as this. 
+The reason we don't do this for RFCs is that they span packages (and as is in the case of this RFC, sometimes span projects). 
+3. [Future] We should also (though can be done later) re-organize the packages folder by suite package name. Using the same names `react`, `react-components`, `react-northstar`, `web-components`.
```

Comment:
Can you elaborate on "future" here? It seems like this is really the core of the fix to address the large scale confusion, but it seems like it's not core to the current RFC as it stands today.

Reply:
Based on the feedback from Jason Falk the main issue was rfcs and specs, reorganizing packages was a nice to have. We can follow with an RFC for package folder restructure, probably not that hard to spec something out, but I think we should move on the easy changes (rfcs and specs) first.

Reply:
> We can follow with an RFC for package folder restructure, 

yes please - https://github.com/microsoft/fluentui/pull/20831/files#r759371376